qiskit_experiments/randomized_benchmarking/rb_utils.py

```diff
@@ -0,0 +1,261 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019-2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+RB Helper functions
+"""
+
+from typing import Tuple, Dict, Optional, Iterable, List
+import numpy as np
+from qiskit import QiskitError, QuantumCircuit
+from qiskit.providers.backend import Backend
+
+
+class RBUtils:
```

Comment:
Why this need to be a class? Looks like this is collection of functions.

Reply:
We might change it since it may be better to put those functions inside of the experiment/analysis classes. So far I'm trying to get the Ignis format to work before committing to more structural changes.

Reply:
You mean adding them to `RBAnalysis` as methods? I think this means all RB subclasses inherit those functionality, even though they don't use, i.e. IRB doesn't need to calculate EPG because we can directly estimate it.